packages/client/src/index.ts

```diff
@@ -1 +1,11 @@
-export default {};
+import {Property, Signature} from "../../message/src/index";
+
+// TODO: @ts-ignore should be removed, when below function is implemented.
+// @ts-ignore
+export async function register_info(url: string, properties: Property[]): Promise<Signature> {
+  // TODO: Not implemented yet.
+}
+
+export function store_signature(properties: Property[], sign: Signature) {
```

Comment:
Only `export` was used instead of `export default`. Do you have any problems?

Reply:
I don't get what do you mean by that.
Is there any specific reason for not using `export default`?